docs/install_guides/installing-openvino-macos.md

```diff
@@ -1,212 +1,156 @@
-| [Documentation for Pre-Trained Models ](@ref omz_models_group_intel) | Documentation for the pre-trained models available in the [Open Model Zoo repo](https://github.com/openvinotoolkit/open_model_zoo). |
+> **NOTE**: The Intel® Distribution of OpenVINO™ is supported on macOS\* version 10.15.x with Intel® processor-based machines.
 
 ## System Requirements
 
-**Hardware**
+@sphinxdirective
+.. tab:: Operating Systems
 
-> **NOTE**: The current version of the Intel® Distribution of OpenVINO™ toolkit for macOS* supports inference on Intel CPUs and Intel® Neural Compute Stick 2 devices only.
+  macOS\* 10.15
 
-* 6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors 
-* 3rd generation Intel® Xeon® Scalable processor (formerly code named Cooper Lake)
-* Intel® Xeon® Scalable processor (formerly Skylake and Cascade Lake)
-* Intel® Neural Compute Stick 2
+.. tab:: Hardware
 
-**Software Requirements**
+  Optimized for these processors:
 
-* CMake 3.13 or higher
-	+ [Install](https://cmake.org/download/) (choose "macOS 10.13 or later")
-	+ Add `/Applications/CMake.app/Contents/bin` to path (for default install) 
-* Python 3.6 - 3.8
-	+ [Install](https://www.python.org/downloads/mac-osx/) (choose 3.6.x - 3.8.x, not latest)
-	+ Add to path
-* Apple Xcode\* Command Line Tools
-	+ In the terminal, run `xcode-select --install` from any directory
-* (Optional) Apple Xcode\* IDE (not required for OpenVINO, but useful for development)
+  * 6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors 
+  * 3rd generation Intel® Xeon® Scalable processor (formerly code named Cooper Lake)
+  * Intel® Xeon® Scalable processor (formerly Skylake and Cascade Lake)
+  * Intel® Neural Compute Stick 2
+  
+  .. note::
+    The current version of the Intel® Distribution of OpenVINO™ toolkit for macOS* supports inference on Intel CPUs and Intel® Neural Compute Stick 2 devices only.
 
-**Operating Systems**
+.. tab:: Software Requirements
 
-- macOS\* 10.15
+  * `CMake 3.13 or higher <https://cmake.org/download/>`_ (choose "macOS 10.13 or later"). Add `/Applications/CMake.app/Contents/bin` to path (for default install). 
+  * `Python 3.6 - 3.9 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.6.x - 3.9.x). Install and	add to path.
+  * Apple Xcode\* Command Line Tools. In the terminal, run `xcode-select --install` from any directory
+  * (Optional) Apple Xcode\* IDE (not required for OpenVINO, but useful for development)
+
+@endsphinxdirective
 
 ## Overview
 
 This guide provides step-by-step instructions on how to install the Intel® Distribution of OpenVINO™ toolkit for macOS*. The following steps will be covered:
 
 1. <a href="#install-core">Install the Intel® Distribution of OpenVINO™ Toolkit</a>
 2. <a href="#set-the-environment-variables">Configure the Environment</a>
-3. <a href="#model-optimizer">Configure the Model Optimizer</a>
+3. <a href="#model-optimizer">Download additional components (Optional)</a>
```

Comment:
Can we add here information on how to download OpenCV using a script supplied by macOS installer?

@mshabunin to provide instructions.

Reply:
![macOS3](https://user-images.githubusercontent.com/3531525/152183865-878fc854-6442-428a-bb6f-4c4cc700f1e6.gif)